tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "importHelpers": true,
     "noImplicitOverride": true,
     "skipLibCheck": true,
+    "sourceMap": true,
```

Comment:
What do you use `sourceMap` for? We don't add source map files into the build folder and it's not needed for debuging